woocommerce/woocommerce-android
# Issue/5631 more menu badge

Closes: #5631 


### Description
More menu unread product reviews badge: 

- Add a badge with no number on the tab itself 
- Add a numbered badge for each of the items in the hub menu that have unread/pending notifications

Designs TTSzMdVvx5A00PsElOc9tI-fi-422%3A51547: 

<img src="https://user-images.githubusercontent.com/2663464/149289817-c00c77e7-5d09-4bef-a1a0-a75f07da2a4d.png" width="600">

### Testing instructions
The following scenarios should be tested: 

With the app in foreground: 
- Add a couple reviews and check the badge is updated accordingly
- "Read" those reviews individually and check that the badge is decremented accordingly for each review read. 
- Add another couple of reviews and the open review list and mark al reviews as read. Wait until the toast that dislays "Reviews succesfully marked as read" appears on the screen. It is a slow request and takes a few secs. I think we should add a spinner for that. 
- Go back and check that badge is gone after marking everything as read. 

### Images/gif

https://user-images.githubusercontent.com/2663464/151952040-e90a637e-7508-4262-9477-02445357075c.mp4
